v1/pkg/client/model_host.go

```diff
@@ -78,4 +78,6 @@ type Host struct {
 	SummaryStatus HealthStatus `json:"SummaryStatus"`
 	// The map of label name to label value for the resource.
 	Labels map[string]string `json:"Labels"`
+	// FC host port world wide port numbers
```

Comment:
Same for the other files below.
```suggestion
	// FC host port world wide port names
```

Reply:
Hi @mchuang3, this is fixed as part of auto generation of code after making changes in host.yaml file.